includes/class-wc-emails.php

```diff
@@ -289,7 +289,7 @@ public function replace_placeholders( $string ) {
 		return str_replace(
 			array(
 				'{site_title}',
-				'{site_address}',
```

Comment:
We can't just remove it from here, we need to keep both, so we avoid any backwards compatibility problem.

Reply:
Fixed, do you think a version check would be appropriate or overkill? Same question for https://github.com/woocommerce/woocommerce/pull/25630/files/89f07da20dc194bfd6ed719820d80db0d39dcf41#diff-176d516db33c360b187bf2539bb465c5